plugins/item_previews/web_client/views/ItemPreviewWidget.js

```diff
@@ -56,6 +56,11 @@ var ItemPreviewWidget = View.extend({
     render: function () {
         var supportedItems = this.collection.filter(this._isSupportedItem.bind(this));
```

Comment:
We should probably only do this logic if `!this.wrapPreviews` to avoid doing computation.